src/java.base/share/classes/java/net/URL.java

```diff
@@ -32,22 +32,18 @@
 import java.nio.file.Path;
 import java.security.AccessController;
 import java.security.PrivilegedAction;
-import java.util.Hashtable;
+import java.util.*;
```

Comment:
Could we avoid the wildcards? Thanks!

Reply:
Sorry for the noise - need to re-tame my IDE after getting set up on a new laptop..